frontend/src/metabase/nav/components/search/RecentsList/RecentsListContent.tsx

```diff
@@ -85,7 +85,7 @@ export const RecentsListContent = ({
                   >
                     {getItemName(item)}
                   </ResultTitle>
-                  <ModerationIcon
+                  <PLUGIN_MODERATION.ModerationStatusIcon
```

Comment:
why did we make this change?

Reply:
For some reason, the plugin object was getting destructured before the ee stuff was finished overwriting things. I couldn't track down the root cause though :/